opennlp-tools/src/main/java/opennlp/tools/util/Cache.java

```diff
@@ -25,6 +25,7 @@
  */
 public class Cache<K,V> extends LinkedHashMap<K,V> {
 
+  private static final long serialVersionUID = 3037039043085908062L;
```

Comment:
Lets not add serialVersionUID, the default is fine, and if we need to change it we can do it.